### ISSUE TITLE
We need to specify the templates that need to be inspected

### ISSUE THREAD

**Issue by svenvandescheur** [sergei-maertens/django-systemjs#11]
Inspected all the templates is really slow. We need a way to indicate what template should be inspected

**Comment by svenvandescheur** [comment]
such very speed

**Comment by sergei-maertens** [comment]
Implemented in #12, example usage:

```
(ima) ➜  imadb git:(develop) ✗ date; ./manage.py systemjs_bundle --template projects/projects_base.html --no-post-process ; date
Wed Apr 27 00:44:15 CEST 2016
Bundled imtool/projects/app.js into SYSTEMJS/imtool/projects/app.js
Wed Apr 27 00:44:22 CEST 2016
```
Will look into more speed gains in the future.

I'm going to take a look if #10 is viable to implement for a 1.4 release, that might block it, but you can run off `develop` for the time being.
